packages/flutter_test/lib/src/mock_canvas.dart

```diff
@@ -497,8 +506,8 @@ class _PathMatcher extends Matcher {
   }
 }
 
-class _MismatchedCall {
-  const _MismatchedCall(this.message, this.callIntroduction, this.call);
+class _MismatchedCall extends Error {
+  _MismatchedCall(this.message, this.callIntroduction, this.call);
```

Comment:
This came from an analysis failure: https://dart.dev/tools/linter-rules/only_throw_errors

Extending Error resolved the failure without requiring any refactoring, which is something I was trying to avoid as much as possible here so as to not introduce a regression. :)

Reply:
This is also why I wrapped all of the `throw '...'` below in `FlutterError`s.

Reply:
Awesome, thanks for clarifying!